rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/360_date_histogram.yml

```diff
@@ -312,3 +312,47 @@ setup:
   - match: { aggregations.date_histogram_no__tz.buckets.2.key: 1619906400000 }
   - match: { aggregations.date_histogram_no__tz.buckets.3.key: 1619910000000 }
 
+---
+"Multi-value date histogram":
+  - skip:
+      version: " - 8.1.99"
+      reason:  Bug fixed in 8.2.0
```

Comment:
Are the skip versions the real fix for the test failures?

Reply:
Yes. These tests were added by PR #84535. As you can see the skip version is the same as it was in the original PR.

Reply:
As a result of PR #85540 these test will not be in 7.17 anymore but they will be in master (and in branch 8.2 created yesterday). Then, if I understand correctly, when executing master v7 compatibility tests they will not be there because they have been removed from 7.17.

Reply:
gotcha